planner/core/plan_test.go

```diff
@@ -232,3 +232,74 @@ func compareStringSlice(c *C, ss1, ss2 []string) {
 		c.Assert(s, Equals, ss2[i])
 	}
 }
+
+func (s *testPlanNormalize) TestNthPlanHint(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists tt")
+	tk.MustExec("create table tt (a int,b int, index(a), index(b));")
+	tk.MustExec("insert into tt values (1, 1), (2, 2), (3, 4)")
+
+	// Test the explained plans.
+	tk.MustQuery("explain select /*+nth_plan(1)*/ * from tt where a=1 and b=1;").Check(testkit.Rows(
+		"TableReader_18 0.01 root  data:Selection_17",
```

Comment:
please write the execution plan test in a son format, so that we can update the result conveniently in the future.